app/src/main/java/com/realityexpander/tasky/agenda_feature/presentation/reminder_screen/ReminderScreen.kt

```diff
@@ -416,44 +417,45 @@ fun ReminderScreenContent(
                 )
                 SmallHeightHorizontalDivider()
 
-                Spacer(modifier = Modifier.largeHeight())
+//                Spacer(modifier = Modifier.largeHeight())
+            }
+            Spacer(modifier = Modifier.weight(1.0f))
 
-                // • DELETE REMINDER BUTTON
-                fun ShowAlertDialogActionType.getTitle() = UiText.Res(
-                        R.string.confirm_action_dialog_title_phrase,
-                        context.getStringSafe(title.asResIdOrNull),
-                        context.getString(R.string.agenda_item_type_reminder)
-                    )
-                fun ShowAlertDialogActionType.getMessage() = UiText.Res(
-                        R.string.confirm_action_dialog_text_phrase,
-                        context.getStringSafe(title.asResIdOrNull).lowercase(),
-                        context.getString(R.string.agenda_item_type_reminder).lowercase()
-                    )
-                TextButton(
-                    onClick = {
-                            onAction(ShowAlertDialog(
-                                title = ShowAlertDialogActionType.DeleteReminder.getTitle(),
-                                message = ShowAlertDialogActionType.DeleteReminder.getMessage(),
-                                confirmButtonLabel =  ShowAlertDialogActionType.DeleteReminder.title,
-                                onConfirm = {
-                                    onAction(DeleteReminder)
-                                }
-                            ))
-                    },
+            // • DELETE REMINDER BUTTON
+            fun ShowAlertDialogActionType.getTitle() = UiText.Res(
+                R.string.confirm_action_dialog_title_phrase,
+                context.getStringSafe(title.asResIdOrNull),
+                context.getString(R.string.agenda_item_type_reminder)
+            )
+            fun ShowAlertDialogActionType.getMessage() = UiText.Res(
+                R.string.confirm_action_dialog_text_phrase,
+                context.getStringSafe(title.asResIdOrNull).lowercase(),
+                context.getString(R.string.agenda_item_type_reminder).lowercase()
+            )
```

Comment:
Why are these functions and not simple variables?

Reply:
I was using them in 2 places at one point, but now there is no need for it in two places, so I will refactor these as simple vars.